test/performance/user_test.rb

```diff
@@ -0,0 +1,42 @@
+require 'test_helper'
+
+require 'benchmark'
+
+module Performance
```

Comment:
I think these changes also exist in a different PR / Branch

Reply:
Yup, wanted the user speedup changes so that tests would run faste

Reply:
If everything is good in this PR, I'll remove this code so that it can be merged